migration/v0.7.x-v0.8.x/migrate-elasticsearch-exporter.md

```diff
@@ -0,0 +1,9 @@
+The `elasticsearch-exporter` chart has been replaced with `prometheus-elasticsearch-exporter`. When upgrading, you must uninstall the old chart before applying the new one.
+
+```
+# Delete old release
+./bin/ck8s ops helm sc uninstall elasticsearch-exporter -n elastic-system
```

Comment:
Why use helm here instead of healmfile? Is it because the old chart is already missing from the helmfile?

Reply:
Yes, the old relase state has been removed from the apps helmfile. `elasticsearch-exporter -> prometheus-elasticsearch-exporter`

Reply:
Yes, that's right (because it changed name). If we change the name back to `elasticsearch-exporter` as it was before, it should be possible to do it with helmfile.

Reply:
Actually, it might not be possible to do it with helmfile even if the release name is changed, since the chart name still differs. Not sure.